ql/cashflows/overnightindexedcoupon.hpp

```diff
@@ -44,6 +44,9 @@ namespace QuantLib {
         rather by the OISRateHelper which is safe, since it reinitialises the
         instrument each time the evaluation date changes.
     */
+
+    enum class OvernightAveraging { Simple, Compound };
+
```

Comment:
This will steal the comment above from the class below — in the sense that Doxygen will associate it with the enum in the generated docs.  It should go before the comment (possibly with a comment of its own?)

Reply:
No problem, I'll add that.